scripts/notifications.coffee

```diff
@@ -0,0 +1,72 @@
+# Description:
+#   Notifications for botgt
+# Commands:
+#   hubot notify <message> - notify everyone with something
+#   hubot notify "@group1 ... @groupn" "#medium1 ... #mediumn" <message>  - notify certain channels with something channels and mediums are optional but order matters (do not mix channels and mediums)
```

Comment:
Remove quotes here, people might think they are necessary, also make `#` the prefix for slack channels since it would be confusing for that to not be the case (rename `group` here to `channel` too). For mediums the prefix can be `@` (but that might be confusing since slack) so maybe `!`? It doesn't matter as long as it doesn't clash.